### PR TITLE
Fix deepsource issues

### PR DESCRIPTION
**Pull request by dITySoftware**

### CODE REVIEW

File: aecid-testsuite/unit/events/DefaultMailNotificationEventHandlerTest.py [hunk 1]
```diff
@@ -26,102 +26,106 @@ class DefaultMailNotificationEventHandlerTest(TestBase):
...
+      log_atom = LogAtom(fixed_dme.fixed_data, ParserMatch(match_element), t, self)
+      default_mail_notification_event_handler.receive_event(self.test % self.__class__.__name__,
           'New value for pathes %s: %s' % ('match/s3',
-          repr(self.match_element.match_object)), [self.log_atom.raw_data], None, self.log_atom, self)
+          repr(match_element.match_object)), [log_atom.raw_data], None, log_atom, self)
 
       t = 0
-      self.default_mail_notification_event_handler.do_timer(t)
+      default_mail_notification_event_handler.do_timer(t)
+      # skipcq: PYL-W1510
```
Comment (landauermax) [inline aecid-testsuite/unit/events/DefaultMailNotificationEventHandlerTest.py:92]: I think this issue can be avoided by adding check=True as a parameter in subprocess.run(). I assume the subprocess should never exit with a non-zero exit code? Please verify that this fix works as intended.

File: aecid-testsuite/unit/events/DefaultMailNotificationEventHandlerTest.py [hunk 2]
```diff
@@ -26,102 +26,106 @@ class DefaultMailNotificationEventHandlerTest(TestBase):
...
         "\n\n") in str(result.stdout, 'utf-8'))
 
       t = time()
-      self.default_mail_notification_event_handler.next_alert_time = t + 500
-      self.default_mail_notification_event_handler.do_timer(t)
+      default_mail_notification_event_handler.next_alert_time = t + 500
+      default_mail_notification_event_handler.do_timer(t)
 
       sleep(5)
+      # skipcq: PYL-W1510
```
Comment (landauermax) [inline aecid-testsuite/unit/events/DefaultMailNotificationEventHandlerTest.py:106]: Same as above

File: aecid-testsuite/unit/events/DefaultMailNotificationEventHandlerTest.py [hunk 3]
```diff
@@ -26,102 +26,106 @@ class DefaultMailNotificationEventHandlerTest(TestBase):
...
+        self.__class__.__name__, description, 1, match_element.get_match_string().decode("utf-8") +
         "\n\n") in str(result.stdout, 'utf-8'))
 
-      self.default_mail_notification_event_handler.next_alert_time = t
-      self.default_mail_notification_event_handler.do_timer(t)
+      default_mail_notification_event_handler.next_alert_time = t
+      default_mail_notification_event_handler.do_timer(t)
 
       sleep(30)
+      # skipcq: PYL-W1510
```
Comment (landauermax) [inline aecid-testsuite/unit/events/DefaultMailNotificationEventHandlerTest.py:118]: Same as above

File: source/root/usr/lib/logdata-anomaly-miner/aminer/AMinerRemoteControlExecutionMethods.py [hunk 4]
```diff
@@ -48,13 +49,14 @@ def change_config_property(self, analysis_context, property_name, value):
                 property_name, t)
             return
 
+        # skipcq: PYL-R1714
```
Comment (landauermax) [inline source/root/usr/lib/logdata-anomaly-miner/aminer/AMinerRemoteControlExecutionMethods.py:52]: Change to property_name in [AMinerConfig.KEY_PERSISTENCE_DIR, AMinerConfig.KEY_LOG_SOURCES_LIST]

File: aecid-testsuite/unit/events/DefaultMailNotificationEventHandlerTest.py [hunk 5]
```diff
@@ -26,102 +26,106 @@ class DefaultMailNotificationEventHandlerTest(TestBase):
...
-      self.log_atom = LogAtom(self.fixed_dme.fixed_data, ParserMatch(self.match_element), t, self)
+      log_atom = LogAtom(fixed_dme.fixed_data, ParserMatch(match_element), t, self)
       sleep(10)
-      self.default_mail_notification_event_handler.receive_event(self.test % self.__class__.__name__, \
+      default_mail_notification_event_handler.receive_event(self.test % self.__class__.__name__,
           'New value for pathes %s, %s: %s' % ('match/s1', 'match/s2',
-          repr(self.match_element.match_object)), [self.log_atom.raw_data, self.log_atom.raw_data], None, self.log_atom, self)
+          repr(match_element.match_object)), [log_atom.raw_data, log_atom.raw_data], None, log_atom, self)
       sleep(30)
+      # skipcq: PYL-W1510
```
Comment (landauermax) [inline aecid-testsuite/unit/events/DefaultMailNotificationEventHandlerTest.py:53]: Same as above

File: source/root/usr/lib/logdata-anomaly-miner/aminer/AMinerRemoteControlExecutionMethods.py [hunk 6]
```diff
@@ -225,6 +227,7 @@ def whitelist_event_in_component(self, analysis_context, component_name, event_d
             else:
                 self.REMOTE_CONTROL_RESPONSE += component.whitelist_event("Analysis.%s" % component.__class__.__name__,
                                                                           [component.__class__.__name__], [LogAtom("", None, 1666.0, None), event_data], whitelisting_data)
+        # skipcq: PYL-W0703
```
Comment (landauermax) [inline source/root/usr/lib/logdata-anomaly-miner/aminer/AMinerRemoteControlExecutionMethods.py:230]: Ok

File: source/root/usr/lib/logdata-anomaly-miner/aminer/parsing/DateTimeModelElement.py [hunk 7]
```diff
@@ -19,8 +19,7 @@ class DateTimeModelElement(ModelElementInterface):
   parsed string."""
 
   # skipcq: PYL-W0613
```
Comment (landauermax) [inline source/root/usr/lib/logdata-anomaly-miner/aminer/parsing/DateTimeModelElement.py:21]: Ok, is already mentioned in comment that it is not yet implemented.

File: aecid-testsuite/unit/events/DefaultMailNotificationEventHandlerTest.py (hunk 1 above)
Comment (dITySoftware) [reply aecid-testsuite/unit/events/DefaultMailNotificationEventHandlerTest.py:92]: Yes i have tried that, but the error message is not clear when using check=True. This subprocess can fail if permissions are missing and then the result with check=False is the same with a clearer error message.

I think it is better to skip this issue. Should i leave it or change the code?

File: source/root/usr/lib/logdata-anomaly-miner/aminer/AMinerRemoteControlExecutionMethods.py (hunk 4 above)
Comment (dITySoftware) [reply source/root/usr/lib/logdata-anomaly-miner/aminer/AMinerRemoteControlExecutionMethods.py]: Done.

File: aecid-testsuite/unit/events/DefaultMailNotificationEventHandlerTest.py (hunk 1 above)
Comment (landauermax) [reply aecid-testsuite/unit/events/DefaultMailNotificationEventHandlerTest.py:92]: Ok, lets skip it for now, maybe we find a better solution in the future.

**Pull request merged**